Configurations/Sanitizers.xcconfig

```diff
@@ -99,3 +101,8 @@ WK_LIBFUZZER_OTHER_LDFLAGS_YES = -fsanitize-coverage=$(WK_LIBFUZZER_COVERAGE);
 
 WK_FUZZILLI_OTHER_CFLAGS_YES = -fsanitize-coverage=trace-pc-guard -DENABLE_FUZZILLI=1;
 WK_FUZZILLI_OTHER_LDFLAGS_YES = -fsanitize-coverage=trace-pc-guard;
+
+// Clang Static Analyzer
+
+// These are to make open source clang work when building WebKit.
+WK_STATIC_ANALYZER_WARNING_CFLAGS_YES = -Wno-availability;
```

Comment:
From what I can tell (see my comments in bugzilla), we only need to do this in internal builds. If so, let's change the variable lookup to `$(WK_STATIC_ANALYZER_WARNING_CFLAGS_$(RUN_CLANG_STATIC_ANALYZER)_$(USE_INTERNAL_SDK));` 
and rewrite this to:
```suggestion
WK_STATIC_ANALYZER_WARNING_CFLAGS_YES_YES = -Wno-availability;
```

Reply:
Having second thoughts about landing this as I don't like the idea of turning off `-Wavailability` for build-and-analyze with internal SDKs.

Reply:
I updated this PR to show what I would have landed, but I'm going to close this as I've changed my mind and I don't think this should land since it's basically an unsupported configuration.

(We may want to add static-analyzer-specific warning flags in the future, so this can be used as a template.)